guides/release/pages.yml

```diff
@@ -48,6 +50,10 @@
   pages:
     - title: "Objects in Ember"
```

Comment:
this can be removed from the Table of Contents, just don't remove the file themselves.

Reply:
Gotcha